pkg/config/spec.go

```diff
@@ -538,6 +540,18 @@ func ReadUkiInstallSpecFromConfig(c *Config) (*v1.InstallUkiSpec, error) {
 func NewUkiUpgradeSpec(cfg *Config) (*v1.UpgradeUkiSpec, error) {
 	spec := &v1.UpgradeUkiSpec{}
 	err := unmarshallFullSpec(cfg, "upgrade", spec)
+	// TODO: Use this everywhere?
+	cfg.Logger.Infof("Checking if OCI image %s exists", spec.Active.Source.Value())
```

Comment:
I tested this and it seems to work, I tihnk we should apply this everywhere, maybe when we construct the source if we find its a docker image we do the check to see if the tag exists. This is very nice because it would be done really soon, while on a install for example, a lot of things are done before we check the tag.

Could probabyl move to to sdk along the image utils.

Something similar for other sources can be done